Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Events.cs

```diff
@@ -38,160 +32,6 @@ public partial class Menu
 
         private bool AllowTooltip => (bool)GetValue(AllowTooltipProperty);
 
-        private static bool NavigateUsingKeyboard(object element, KeyEventArgs args, Menu menu, Orientation orientation)
```

Comment:
It was easier to search where are events and where are helper methods. the file was too big I should have planned the logic well from the beginning but we added many features during the PR :)